File: app/Models/Task.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Task extends Model
{
    use HasFactory;

    // 白名单
    protected $fillable = [
        'job_id',
        'name',
        'shell',
        'log',
        'status',
        'created_at',
        'updated_at',
    ];
}

File: app/models/task.go
package models

import "github.com/goravel/framework/database/orm"

const (
	TaskStatusWaiting = "waiting"
	TaskStatusRunning = "running"
	TaskStatusSuccess = "finished"
	TaskStatusFailed  = "failed"
)

type Task struct {
	orm.Model
	Name   string `gorm:"not null;index" json:"name"`
	Status string `gorm:"not null;default:'waiting'" json:"status"`
	Shell  string `gorm:"not null" json:"shell"`
	Log    string `gorm:"not null" json:"log"`
}
